data/data.py

```diff
@@ -75,11 +77,17 @@ def set_current_date(self):
   # mlbgame refresh
 
   def refresh_standings(self):
+    # TODO: This function is a workaround for MLB and mlbgame package no longer supporting GameDay XML API
+    # Old code is left for reference only!
+
+
     try:
-      if self.config.demo_date:
-        self.standings = mlbgame.standings(datetime(self.year, self.month, self.day, 23, 59, 0, 0))
-      else:
-        self.standings = mlbgame.standings()
+      # if self.config.demo_date:
```

Comment:
Can you get rid of the comments here? The if check appears redundant anyways so just scrap it.

Reply:
Sure thing, taken care of.